src/compiler.adoc

```diff
@@ -881,10 +881,8 @@ with the following content:
 ----
 
 As you can observe, the code now looks more “Clojure-friendly.” The host
-interoperabilty calls are almost hidden.
-
-NOTE: This example does not means that we prefer `dommy` over the Google Closure library. In
-fact, dommy does not have documentation, so we do not recommend use it.
+interoperabilty calls have now been replaced by functions provided by this
+third-party library.
```

Comment:
- rephrase: hidden -> replaced with ... (just concludes the section by fully stating what we accomplished)
- opinion: removed opinion on dommy, thinking about impartiality